src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -762,8 +762,13 @@ public final ConveyorTile dispatch(
         }
 
         if (!tld.layerExists(layerName)) {
-            requestMistmatchTarget.append("not a tile layer");
-            return null;
+            // GEOS-9431-check in catalog also if not found TileLayerDispatcher
+            LayerInfo layerInfo = catalog.getLayerByName(layerName);
```

Comment:
This needs to vouch for layer groups.

Reply:
understood,will do, make a separate method to looks for the name as layer or layergroup